Makefile

```diff
@@ -2,10 +2,10 @@ build-dev:
 	docker build -t openslides-client-dev -f Dockerfile.dev .
 
 run-dev: | build-dev
-	docker run -t -v `pwd`/client:/app -p 127.0.0.1:4200:4200/tcp openslides-client-dev
+	docker run -ti -v `pwd`/client:/app -p 127.0.0.1:4200:9001/tcp openslides-client-dev
```

Comment:
please also change the port in the `package.json`

Reply:
I thought the port 9001 was needed for the full setup? I specifically only touched this to be able to access it. If you want, I can set the host port to 9001, too.

Reply:
Yes, then it is consistent at all places